scripts/janusbase.js

```diff
@@ -1,6 +1,6 @@
 elation.require(['engine.things.generic', 'utils.template'], function() {
   elation.template.add('janusweb.edit.object', 
-      '<Object id=^{id}^ js_id=^{js_id}^ locked=^false^ pos=^{pos.x} {pos.y} {pos.z}^ vel=^{vel.x} {vel.y} {vel.z}^ accel=^{accel.x} {accel.y} {accel.z}^ xdir=^{xdir}^ ydir=^{ydir}^ zdir=^{zdir}^ scale=^{scale.x} {scale.y} {scale.z}^ col=^{col}^ lighting=^{lighting}^ visible=^{visible}^ />');
+      '<Object id=^{id}^ js_id=^{js_id}^ alphatest=^{alphatest}^ locked=^false^ pos=^{pos.x} {pos.y} {pos.z}^ vel=^{vel.x} {vel.y} {vel.z}^ accel=^{accel.x} {accel.y} {accel.z}^ xdir=^{xdir}^ ydir=^{ydir}^ zdir=^{zdir}^ scale=^{scale.x} {scale.y} {scale.z}^ col=^{col}^ lighting=^{lighting}^ visible=^{visible}^ />');
```

Comment:
This template is actually only used by a version of the room source viewer which is currently disabled.  This is basically room serialization code, but it needs to be extended to work with other kinds of objects, it's currently very hacky.

The only place it's called is in room.getRoomSource() - https://github.com/jbaicoianu/janusweb/blob/master/scripts/room.js#L1694 - what's really needed is for each class which extends the janusbase class to also override the summarizeXML() function, rather than having one template with all attributes and then hacking up the object type later.